### ISSUE TITLE
Camera always start from (0,0,0) position.

### ISSUE THREAD

**Issue by AzadNezamdoust** [provencher/MRTK-Quest#99]
Camera always start from (0,0,0) position. In xr interaction toolkit it is based on the local position of the parent gameobject of the camera but here it s overriden by the main position so I can't move user to start from a specific point of the scene.
I could n't find a way to do it. Also I want to create snap teleportation points so i need some control over it.

**Comment by provencher** [comment]
Have you tried moving the playspace root?

**Comment by AzadNezamdoust** [comment]
Yes I did at start of the experience camera moves to the 0,0,0 in world space and ignores the plays pace game object transform.

**Comment by AzadNezamdoust** [comment]
> Have you tried moving the playspace root?

Yes I did at start of the experience camera moves to the 0,0,0 in world space and ignores the playspace game object transform.

**Comment by AzadNezamdoust** [comment]
> Have you tried moving the playspace root?

Help please.
